src/index.ts

```
@@ -36,6 +44,11 @@ export const sentry = <Context>({
   return async function(resolve, parent, args, ctx, info) {
     try {
       const res = await resolve(parent, args, ctx, info)
+
+      if (reportError && reportError(res)) {
```

Wouldn't that check have to be `!reportError(res)` and return the error, rather than returning if it should be reported?

---

Good catch, I fixed this and added a line explaining it for clarification